app/views/menu/menu.html.erb

```diff
@@ -110,11 +115,11 @@
                         if (dish.star == 0) 
                         ""
                         elsif (dish.star == 1) 
-                        "*"
+                        "Dinner"
                         elsif (dish.star == 2) 
-                        "**"
+                        "Supper"
                         else 
-                        "*/**"
+                        "D + S"
```

Comment:
This is an improvement, as it is now readable, but I still think that ANY information on preset meals is unnecessary to show to non-staff users. (People who would be getting the preset meals won't even be looking at the menu anyway)

Let me know why (on slack?) you think this should be visible to non-staff if you are confident that the code should stay the way it is.

Reply:
I agree! I forgot to change it, give me one sec.